generator.yaml

```diff
@@ -8,6 +8,10 @@ operations:
 resources:
   Table:
     fields:
+      TimeToLive:
```

Comment:
I'm almost tempted to say let's call the field "TTL", since that acronym is so common... but cool to spell it out I guess :)